src/app/own-leave/leave/leave.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'pm-leave',
+  templateUrl: './leave.component.html',
+  styleUrls: ['./leave.component.css']
+})
+export class LeaveComponent implements OnInit {
+
+  constructor() { }
```

Comment:
jak nie potrzebujesz to chyba lepiej nie dawac pustego konstruktora, to samo z implementacja interfejsu